src/index.js

```diff
@@ -1,6 +1,6 @@
 import './init';
 import React from 'react';
-import ReactDOM from 'react-dom';
+import ReactDOM from 'react-dom/client';
```

Comment:
We can remove this. I've not set the `react` and `react-dom` versions to `18` in `package.json`.